pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -93,7 +93,6 @@ var _ = Describe("Pod Network", func() {
 
 		ctrl = gomock.NewController(GinkgoT())
 		mockNetwork = NewMockNetworkHandler(ctrl)
-		Handler = mockNetwork
```

Comment:
shouldn't this be part of the next commit  ?

Reply:
This commit remove the "Handler" variable so we have to remove it from the test too

Reply:
Of course. Thanks.